### PR TITLE
refactor(core): no need to await for publish as it can be done asynchronously

### PR DESCRIPTION
**Pull request by machi1990**

### CODE REVIEW

**Comment by wtrocki** [comment]
**So the current situation wasn't good.** 
If there is a problem with the network connection client would got error (despite proper save).

**After the change we get:**
Save will happen and user will get a positive result but subscriptions might not get it. 

I think this is good behavior/improvement. Verifying to be sure

**Comment by machi1990** [comment]
> **Current situation**
> If there is a problem with the network connection to queue client would got an error (despite proper save to db) client cannot react or handle this error (as it is related to server side problem).
> 

Exactly. Any publish error would have resulted in failure of the whole request even if the save / update / delete operations went well. 

> **After the change we get:**
> Save will happen and user will get a positive result but subscriptions might not get it.
> Logging should be improved (but that is actually an template side code.

+1. I think a `.then()` and `.catch()` as we can have appropriate logs for each case but went with `.finally()` and we can add a log message here. I can create a "good first issue" for that. 


> I think this is good behavior/improvement. Verifying to be sure

Thanks.

File: packages/graphback-core/src/runtime/CRUDService.ts [hunk 1]
```diff
@@ -55,7 +55,8 @@ export class CRUDService<Type = any> implements GraphbackCRUDService<Type>  {
       const topic = this.subscriptionTopicMapping(GraphbackOperationType.CREATE, this.model.graphqlType.name);
       //TODO use subscription name mapping
       const payload = this.buildEventPayload('new', result);
-      await this.pubSub.publish(topic, payload);
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      this.pubSub.publish(topic, payload).finally(() => {});
```
Comment (wtrocki) [inline packages/graphback-core/src/runtime/CRUDService.ts:59]: ok. Havent's seen that. Why finnally is empty? Makes no sense

**Comment by wtrocki** [comment]
Approved and then seen your comment.

> +1. I think a .then() and .catch() as we can have appropriate logs for each case but went with .finally() and we can add a log message here. I can create a "good first issue" for that.

Is there any problem with logging that prevents us from this

File: packages/graphback-core/src/runtime/CRUDService.ts (hunk 1 above)
Comment (machi1990) [reply packages/graphback-core/src/runtime/CRUDService.ts:59]: We can surely add logs, I was just too lazy to add them.

Comment (machi1990) [reply packages/graphback-core/src/runtime/CRUDService.ts]: I've added a console.error on this, so that users get's to see what went wrong.

**Comment by wtrocki** [comment]
All good for me

**Pull request merged**